aws/sdk/examples/lambda-invoke-function/src/main.rs

```diff
@@ -2,52 +2,139 @@
  * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  * SPDX-License-Identifier: Apache-2.0.
  */
+ 
+//! Minimal example that invokes an AWS Lambda function
+//!
+//! Invokes a function and prints the response without attempting to
+//! deserialize the returned data.
 
-use std::process;
+/// types from the Rust standard library
```

Comment:
can you run `cargo doc` on this? It might not like applying `///` comments to use statements

Reply:
checking

Reply:
Running `cargo doc` only generated on warning and it was on the link not being in the `<url>` format